src/index.html

```diff
@@ -2,13 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="header-img">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="header-nav">
+        <ul class="header-items">
```

Comment:
```suggestion
        <ul class="nav__items">
```
After you create nav as a separate block them rename all the elements and use the double underscore for naming

Reply:
I agree, but this error occurs:
34:1  ✖  Expected class selector ".nav__img" to be kebab-case    selector-class-pattern